style.css

```diff
@@ -96,3 +96,272 @@ a {
   background-position-x: right;
   background-position-y: 120px;
 }
+
+.recent-work {
+  width: 375px;
+  height: 6080px;
+}
+
+.top {
+  width: 327px;
+  height: 107px;
+  margin-top: 120px;
+  margin-left: 24px;
+  margin-right: 24px;
+  color: #1c1a19;
+}
+
+.top h4 {
+  font-family: crete Round, sans-serif;
+  font-size: 30px;
+  line-height: 52px;
+  text-align: left;
+  vertical-align: top;
+  color: #1c1a19;
+}
+
+.fram-two {
+  margin-top: 114px;
+  width: 375px;
+  height: 568px;
+  margin-left: 24px;
+}
+
+.featured-image img {
+  width: 327px;
+  height: 237px;
+}
+
+.right-block {
+  background-color: #e5e5e5;
+  width: 350px;
+  height: 320px;
+  padding-bottom: 300px;
+}
+
+.right-block h6 {
+  line-height: 24px;
+  font-family: Crete Round, sans-serif;
+  font-size: 18px;
+  padding-top: 10px;
+  color: #091e42;
+  text-align: center;
+  vertical-align: top;
+}
+
+.right-block p {
+  font-size: 12px;
+  font-family: inter, sans-serif;
+  line-height: 16px;
+  color: #091e42;
+  padding-left: 0;
+  margin-right: 40px;
+  padding-right: 24px;
+}
+
+.tags {
+  width: 297px;
+  height: 40px;
+  gap: 12px;
+}
+
+.tags img {
+  width: 49px;
+  height: 40px;
+  gap: 30px;
+  flex-direction: row;
+  padding-top: 20px;
+  padding-bottom: 18px;
+}
+
+.tags button {
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 3%;
+  color: #fff;
+}
+
+.frame43 {
+  width: 375px;
+  height: 2436px;
+  background-color: #fff;
+}
+
+.mask-group-one {
+  width: 327px;
+  height: 110px;
+  background-image: url("image/imgPlaceholderTwo.png");
+  background-repeat: no-repeat;
+  margin-left: 24px;
+  padding-bottom: 386px;
+  background-position-x: right;
+  background-position-y: 120px;
+}
```

Comment:
- [ ] Please use the CSS Flex box to align contents in each of your cards. This helps you structure content in cards well and it's required you used that to align-items or contents within your project cards.
- [ ] Kindly adjust the padding of the description text for each of your cards and buttons. The badges have no margin from the left. Please try to make your design look like the one given in the Figma design.
- [ ] Please compare the two screenshots in the comment, I marked red on your and green on the Figma design so you see the points you need to work on.

Your Design
![image](https://user-images.githubusercontent.com/50941074/160246198-c5999538-f10e-43fa-a8ad-0a18af1dae27.png)

The Figma Design
![image](https://user-images.githubusercontent.com/50941074/160246284-5b46035c-a34a-43df-b606-2e811b916662.png)

Reply:
resolved 👍